app/components/settings/rules/actions_form_row_component.html.erb

```diff
@@ -3,9 +3,13 @@
     <%= @form.hidden_field :id %>
     <div class="flex gap-1">
       <p class="text-base font-medium text-left text-gray-900"><%= t @form.object.type %></p>
-      <p class="text-base text-left text-gray-900"><%= @form.object.action_object.name %></p>
+      <% if @form.object.type == "Automation::AddMessageThreadTagAction" %>
+        <p class="text-base text-left text-gray-900"><%= @form.object.action_object.name %></p>
```

Comment:
Toto dratovanie na konkretnu triedu sa mi nepaci. Preco tu potrebujeme hidden field a inde nie?

Reply:
V com je ta akcia ina?

Reply:
Zasadne odlisenie tych akcii je, ze jedna je parametrizovana hodnotou (t.j. vies tam napisat hocico) a druha vyberom tagu - t.j. vyberas tag.

Reply:
v tom pripade by som sa radsej opytal objektu nejaky `has_name?` alebo nieco co dava zmysel a nedratoval to na konstantu. lebo ked pridame dalsiu triedu tak tu to bude treba menit.